config/allscripts.yml

```diff
@@ -11,11 +11,12 @@ auth:
   scope: launch/patient patient/*.read offline_access
   confidential_client: True
   steps:
-    - find: "[translate='DEFAULT_LOGIN_BUTTON']"
-    - click
-    - find: "#UserName"
-    - type: "s4s_6-22-16"
-    - find: "#Password"
-    - type: "s4s!2345"
-    - find: "[translate='Login_LogIn']"
-    - click
+    - element: "[translate='DEFAULT_LOGIN_BUTTON']"
```

Comment:
I get what you mean about the empty `click:`. I was tempted to fill them with `click: True` or something like that, just as a placeholder. On the other hand, interchanging dictionaries with a single key (`find:`) and scalars (`click`) was terrifying :D